src/main/scala/stdlib/Lists.scala

```diff
@@ -141,9 +141,9 @@ object Lists extends FlatSpec with Matchers with org.scalaexercises.definitions.
 
   /** You can create a list from a range
     */
-  def fromRangeLists(res0: List[Int]) {
+  def fromRangeLists(res0: Int, res1: Int, res2: Int, res3: Int, res4: Int) {
```

Comment:
I assumed it would be rendered in a nicer way on the website "should be(List(...))" instead of "should be(...)"